Demo/Demo/SwiftUIComponents/PayPalVaultViews/PayPalVaultResultView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import PayPalWebPayments
+
+struct PayPalVaultResultView: View {
+
+    @ObservedObject var paypalVaultViewModel: PayPalVaultViewModel
```

Comment:
```suggestion
    @ObservedObject var viewModel: PayPalVaultViewModel
```
🧶 : Similar to Rich's callout, since the `var` is strongly typed here we can [omit type](https://www.swift.org/documentation/api-design-guidelines/#name-according-to-roles) from the name.

Reply:
Forgot to fix this one on last PR feedback fix commit